storage/rocksdb/ha_rocksdb.cc

```diff
@@ -2474,13 +2472,6 @@ static int rocksdb_prepare(handlerton *const hton, THD *const thd,
       if (!tx->prepare(rdb_xid_to_string(xid))) {
         return HA_EXIT_FAILURE;
       }
-      if (thd->durability_property == HA_IGNORE_DURABILITY &&
```

Comment:
`thd_store_lsn` is a part of facebook/mysql-5.6@a869c56d361bb44f46c0efeb11a8f03561676247